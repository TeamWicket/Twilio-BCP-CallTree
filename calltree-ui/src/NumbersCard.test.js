import NumbersCard from './NumbersCard';
import React from 'react';
import { shallow } from 'enzyme';

describe("Numbers Card", () => {
    it("should match snapshot of Numbers Card Component", () => {
        expect(shallow(<NumbersCard />)).toMatchSnapshot();
    })
});
