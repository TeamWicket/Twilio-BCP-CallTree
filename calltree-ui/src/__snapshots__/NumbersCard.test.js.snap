// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Numbers Card should match snapshot of Numbers Card Component 1`] = `ShallowWrapper {}`;
